_posts/2017-11-23-Rust-1.22.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "Announcing Rust 1.22"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.22.0. Rust
+is a systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.22 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.22.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1220-2017-11-23
+
+## What's in 1.22.0 stable
+
+The headline feature for this release is one many have been waiting for
+for a long time: you can now [use `?` with `Option<T>`]! About a year
+ago, in [Rust 1.13], we introduced the `?` operator for working with
+`Result<T, E>`. Ever since then, there's been discussion about how far
+`?` should go: should it stay only bound to results? Should it be
```

Comment:
"bound" makes me think of "variable bindings", maybe "should it stay only for results?"